### ISSUE TITLE
Assets .htaccess too restrictive in some cases

### ISSUE THREAD

**Issue by axllent** [silverstripe/silverstripe-assets#142]
I fully comprehend the reasoning for the tightened restrictions in the auto-generated `public/assets/.htaccess` file (4.1.1), however this only suits situations whereby the root of the site is loaded from a domain name, eg: `http://mysite.com/assets/...`. When for instance files are loaded from something like: `http://dev.server/mysite.com/assets/...` (think of a general development server etc) access to all files is blocked due to the extra `.` in the url.

Is there maybe a more dynamic way it can generate the htaccess restrictions relative to the website's basedir instead of assuming/enforcing each site has it's own domain?

**Comment by dhensby** [comment]
which rule is blocking the extra `.`? I'd be happy to loosen this kind of restriction if there aren't any exploitable weaknesses by doing so

**Comment by dhensby** [comment]
Actually, I'm going to close this as a duplicate of #141

**Comment by axllent** [comment]
I saw that issue after I finished this one (created roughly the same time). Whilst similar, that one refers to files with two dots. My issue refers to an extra dot in the folder. Different situations being blocked by the same rule.
